lcm-cmake/cpack.cmake

```diff
@@ -0,0 +1,44 @@
+# Package release version
+set(PACKAGE_RELEASE_VERSION 1)
+
+# Default CPack generators
+set(CPACK_GENERATOR "TGZ;STGZ")
+
+# Determine distribution
+execute_process(COMMAND lsb_release -si
+	OUTPUT_VARIABLE LINUX_DISTRO
+	OUTPUT_STRIP_TRAILING_WHITESPACE
+)
+
+# Determine architecture
+execute_process(COMMAND uname -m
+	OUTPUT_VARIABLE	MACHINE_ARCH
```

Comment:
Heh, I didn't even notice the indent :smile:... I was wondering about the space between `OUTPUT_VARIABLE` and `MACHINE_ARCH`. (Which is also fixed; thanks!)